pokemon-league-participation-manager/src/main/java/cz/muni/fi/pa165/pokemon/league/participation/manager/entities/DAOInterface/PokemonSpeciesDAO.java

```diff
@@ -0,0 +1,67 @@
+package cz.muni.fi.pa165.pokemon.league.participation.manager.entities.DAOInterface;
+
+import cz.muni.fi.pa165.pokemon.league.participation.manager.entities.PokemonSpecies;
+import cz.muni.fi.pa165.pokemon.league.participation.manager.common.*;

```

Comment:
Don't use * imports.

Reply:
I think that import is OK, basically I completely agree with not using import *, but in this case it just reduces the number of import lines.

Reply:
Fixed